pkg/rancher-desktop/main/deploymentProfiles.ts

```diff
@@ -209,13 +217,25 @@ class Win32DeploymentReader {
     }
 
     // Don't bother with the validator, because the registry-based reader validates as it reads.
+    if (this.errors.length) {
+      throw new DeploymentProfileError(`Error in registry settings:\n${ this.errors.join('\n') }`);
+    }
+
     return { defaults, locked };
   }
 
   protected fullRegistryPath(...pathParts: string[]): string {
     return `${ this.keyName }\\${ this.registryPathProfile.join('\\') }\\${ pathParts.join('\\') }`;
   }
 
+  protected msgFieldExpectingReceived(field: string, expected: string, received: string) {
+    return `Error for field '${ field }': expecting ${ expected }, got ${ received }`;
+  }
+
+  protected msgFieldExpectingTypeReceived(field: string, expectedType: string, received: string) {
+    return this.msgFieldExpectingReceived(field, `value of type ${ expectedType }`, received);
```

Comment:
I notice that the darwin/linux flow says `an array` instead of `value of type array`.  We should make them match.

Reply:
I don't love that form, but it's more consistent with the other messages.